components/ui/reset-formation.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Button, buttonVariants } from './button';
+
+
+interface ResetFormationProps {
+    onReset: () => void;
+}
+
+const ResetFormation: React.FC<ResetFormationProps> = ({ onReset }) => {
```

Comment:
generally speaking we prefer to just use a `function` instead of a `const`

```
export default function ResetFormation({onReset}: ResetFormationProps) {
  ...
}
```